src/models/train_model.py

```diff
@@ -0,0 +1,55 @@
+import pytorch_lightning as pl
+from torch import save, tensor
+import pandas as pd
+from model import LightningModel
+
+
+# loads the data from the processed data folder
+def load_data():
+    return pd.read_csv('data/processed/data.csv')
+
+
+# trains the lightning model with the data
+def train(data: pd.DataFrame, hparams: dict):
+    # convert data to tensors, where all columns in the dataframe
+    # except TAc are inputs and TAc is the target
+    x = tensor(data.drop(columns=["TAc"]).values).float()
+    y = tensor(data["TAc"].values).float()
+
+    # for every column in the input values, apply a min max normalization
+    for i in range(x.shape[1]):
+        x[:,i] = (x[:,i] - x[:,i].min()) / (x[:,i].max() - x[:,i].min())
+
+    model = LightningModel(hparams)
+
```

Comment:
I would suggest to use some checkpoint here as well

ModelCheckpoint(dirpath="./outputs", 
                                          monitor="train_loss", 
                                          mode="min"
                                          )